docs/BigQueryTable-batchsink.md

```diff
@@ -46,6 +46,10 @@ bucket will be created and then deleted after the run finishes.
 will be dropped.
 * Upsert - records that match on Table Key will be updated. Records that do not match will be inserted.
 
+**Truncate Table**: Whether to truncate the output table before writing to it.
+* True - Will delete all rows in the table before writing to it.
+* False - Will append the new rows to the table.
```

Comment:
It doesn't necessarily append, depending on what is set for 'Operation'. I think we can just remove the true false lines, it seems clear enough with the single sentence description.

Reply:
Fixed by shortening the doc to a single sentence.